examples/jsm/renderers/webgpu/nodes/WGSLNodeBuilder.js

```diff
@@ -207,6 +219,12 @@ class WGSLNodeBuilder extends NodeBuilder {
 
 	}
 
+	getTextureCompare( texture, textureProperty, uvSnippet, shaderStage = this.shaderStage ) {
```

Comment:
Why do we need both `getTextureCompare` and `getSamplerCompare` functions?

(By the way, could we merge `getTexture` with `getTextureLevel` and `getSampler` with `getSamplerLevel`?)